### PR TITLE
chore: updated tasegir

### PR DESCRIPTION
**Pull request by nduchak**

### CODE REVIEW

File: src/rates/update.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export async function updater (): Promise<void> {
           throw Error('Unsupported fiat')
         }
 
-        // @ts-ignore
+        // @ts-ignore: not typed
```
Comment (AuHau) [inline src/rates/update.ts:66]: Heh lazy you 🤪 This does meet the requirement, but not the reason for that requirement 😉 Some better explanation would be better...

File: src/sequelize.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export function sequelizeFactory (): Sequelize {
       return filename.substring(0, filename.indexOf('.model')) === member.toLowerCase()
     },
     logging: (msg) => logger.debug(formatLogs(msg)),
-    // @ts-ignore
+    // @ts-ignore: Not typed
```
Comment (AuHau) [inline src/sequelize.ts:32]:
```suggestion
    // @ts-ignore: Incorrectly typed in the sequelize library
```

File: src/services/storage/handlers/offer.ts [hunk 3]
```diff
@@ -113,7 +113,7 @@ const handler: Handler<StorageOfferEvents, StorageServices> = {
     const provider = event.returnValues.provider
 
     // TODO: Ignored until https://github.com/sequelize/sequelize/pull/11924
-    // @ts-ignore
+    // @ts-ignore: issue ^
```
Comment (AuHau) [inline src/services/storage/handlers/offer.ts:116]:
```suggestion
    // @ts-ignore: TODO: Ignored until https://github.com/sequelize/sequelize/pull/11924 
```

File: src/services/storage/handlers/offer.ts [hunk 4]
```diff
@@ -113,7 +113,7 @@ const handler: Handler<StorageOfferEvents, StorageServices> = {
     const provider = event.returnValues.provider
 
     // TODO: Ignored until https://github.com/sequelize/sequelize/pull/11924
```
Comment (AuHau) [inline src/services/storage/handlers/offer.ts:115]:
```suggestion
```

**Review by AuHau — CHANGES_REQUESTED** [review]
There is a reason for the lint rule for the `@ts-*` directives, putting everywhere "not typed" is meeting the requirement but not the reason for it. If you won't do it now, you will never do it. I can accept few places where are you not sure for it, but putting it everywhere is not a really a good idea. At least comment the ones in `/src`, we can ignore the test's ones...

File: src/rates/update.ts (hunk 1 above)
Comment (nduchak) [reply src/rates/update.ts:66]: Hehe, agree. 
Will fix it.

Comment (nduchak) [reply src/rates/update.ts:66]: Done

**Review by AuHau — APPROVED** [review]
Sweet! LGTM now! Thanks for the explanations.

**Pull request merged**